exposures/logs/cnp-ingress-deny-dotnet-trace-on-public-web.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "cnp-ingress-deny-dotnet-trace-on-public-web"
+  namespace: default          #change default namespace to match your namespace
+spec:
+  description: "Deny the /Trace.axd endpoint by whitelisting all other http rules"
+  endpointSelector:
+    matchLabels:
+      app: dotnet-app    #change app: dotnet-app to match your label
```

Comment:
If it'll be valid for the Dotnet application only?
- Then we can include `dotnet` word in description also